projects/user_publication/gscholar.py

```diff
@@ -0,0 +1,207 @@
+import logging
+import re
+from datetime import datetime
+from pathlib import Path
+
+from scholarly import ProxyGenerator, scholarly
+from scholarly._proxy_generator import MaxTriesExceededException
+
+from projects.models import ChameleonPublication, Publication
+from projects.user_publication import utils
+
+logger = logging.getLogger("projects")
+
+MAX_RETRIES = 10
+
+
+def handle_scrapper():

```

Comment:
the name of this method is a bit confusing, since it's main purpose is to get the API key. how about something like `get_scraper_api_key`?